src/components/Tags/Tags.visual.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import { Tags } from './';
+import { ALIGNMENT, SIZE, SKIN } from './constants';
+import { snap, story, visualize } from 'storybook-snapper';
+import { TPAComponentsProvider } from '../TPAComponentsConfig';
+
+const MANY_ITEMS_COUNT = 20;
+const FEW_ITEMS_COUNT = 3;
+
+function generateItems(amount) {
+  return new Array(amount).fill(null).map((el, i) => ({
+    title: `Title ${i}`,
+    value: `value ${i}`,
+    checked: i % 3 === 0,
+  }));
+}
+
+function alignmentTestSuite(alignment?) {
+  story(`Align ${alignment ? alignment : 'default'}`, () => {
+    snap('many item', () => (
+      <Tags
+        items={generateItems(MANY_ITEMS_COUNT)}
+        onClick={() => {}}
+        alignment={alignment}
+      />
+    ));
+
+    snap('several item', () => (
+      <Tags
+        items={generateItems(FEW_ITEMS_COUNT)}
+        onClick={() => {}}
+        alignment={alignment}
+      />
+    ));
+
+    snap('rtl', () => (
+      <div dir={'rtl'}>
+        <Tags
+          items={generateItems(FEW_ITEMS_COUNT)}
+          onClick={() => {}}
+          alignment={alignment}
+        />
+      </div>
+    ));
+
+    snap('RTL', () => (
+      <TPAComponentsProvider value={{ rtl: true }}>
+        <Tags
+          items={generateItems(FEW_ITEMS_COUNT)}
+          onClick={() => {}}
+          alignment={alignment}
+        />
+      </TPAComponentsProvider>
+    ));
+  });
```

Comment:
Oh and another thing.
You're right, there is a problem with the `rtl` visual tests.
It's not working as expected (you can see it here: https://wixeyes.applitools.com/app/test-results/00000251817847633468/?accountId=mE99eP8CUn9UTSXSv98L3OSaJ0NZmAcY6Wndx6WSHODdY110)

So first, I think there should be one `rtl` test, i'm not sure that applitools is case-sensitive.
And second, the test should both have a `<div dir="rtl">...</div>` and also the `<TPAComponentsProvider>` together.
i.e.:
```jsx
<div dir={'rtl'}>
  <TPAComponentsProvider value={{ rtl: true }}>
    <Tags
      items={generateItems(FEW_ITEMS_COUNT)}
      onClick={() => {}}
      alignment={alignment}
    />
  </TPAComponentsProvider>
</div>
```